### PR TITLE
[analyzer] Disable clang flags that cause major undeterminism issues

### PR DESCRIPTION
**Pull request by Szelethus**
Turn off Z3 timeouts added to the Clang Static Analyzer in https://github.com/llvm/llvm-project/pull/95129. We are now aware that these options can often cause nondeterministic behaviour.

### CODE REVIEW

File: config/config_files/codechecker.json [hunk 1]
```diff
@@ -4,6 +4,10 @@
     "--enable=core.CallAndMessage",
     "--analyzer-config",
     "clangsa:unroll-loops=true",
+    "--analyzer-config",
```
Comment (dkrupp) [inline config/config_files/codechecker.json:7]: isn't this config file just an example? Will it get in effect by default?

**Review by dkrupp — COMMENTED** [review]
isn't this config file just an example? Will it get in effect by default?

File: config/config_files/codechecker.json (hunk 1 above)
Comment (Szelethus) [reply config/config_files/codechecker.json:7]: I initially intended to put the change here, I'm just not sure whether hardcoding such a change is appropriate:
https://github.com/Ericsson/codechecker/blob/master/analyzer/codechecker_analyzer/analyzers/clangsa/analyzer.py#L392
@bruntib?

**Comment by dkrupp** [comment]
I guess this patch can be abandoned... or?

**Comment by Szelethus** [comment]
Tahts right, sorry!

**Pull request closed without merging**